BUILD.gn

```diff
@@ -185,7 +185,9 @@ group("create_dist") {
   if (is_android) {
     deps += [ "build/android:sign_app" ]
   } else {
-    deps += [ ":create_symbols_dist" ]
+    if (brave_debug_symbol_level != 0) {
```

Comment:
why are we adding a new config, I thought we were going to use `is_brave_release_build`?

Reply:
Discussed in DM:
better to create specific param for each feature then use the general root args everywhere, because it's more flexible and readable. Chromium do the same: it mostly uses `is_official_builds` to calculate value of new args then directly in if-statements. Example 1. 2. 3.